tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Kata Containers Tests
+
+This directory contains various types of tests for testing the Kata Containers
+repository.
+
+## Test Content
+
+We provide several tests to ensure Kata-Containers run on different scenarios
+and with different container managers.
+
+1. Integration tests to ensure compatibility with:
+   - [Kubernetes](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/kubernetes)
+   - [Cri-Containerd](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/cri-containerd)
+   - [Docker](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/docker)
+   - [Nerdctl](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/nerdctl)
+   - [Nydus](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/nydus)
+   - [Runk](https://github.com/kata-containers/kata-containers/tree/main/tests/integration/runk)
+2. [Stability tests](https://github.com/kata-containers/kata-containers/tree/main/tests/stability)
+3. [Metrics](https://github.com/kata-containers/kata-containers/tree/main/tests/metrics)
+4. [Functional](https://github.com/kata-containers/kata-containers/tree/main/tests/functional)
+
+## GitHub Actions
+
+Kata Containers uses GitHub Actions in the [Kata Containers](https://github.com/kata-containers/kata-containers) repository.
+
+## Breaking Compatibility
```

Comment:
I don't think this section is relevant now we have a single repo for dev and test code, so don't need the depends on links now?

Reply:
@stevenhorsman I remembered that the Depends-on also work even if the PR was also in the same repository, it was mainly used when 2 people are working separately and they want to test their changes, but it is ok for me if you want to remove this

Reply:
My understanding (which is probably wrong) is the the logic for processing the `Depends-on` tag was just in `resolve-kata-dependencies.sh` that was in the `tests` repo, so even if there is potentially a use case for have two PRs in the same repo where the tests are done independently from the dev code) (e.g. the attestation tests at the moment). I don't think we have anything that processing this logic anymore and it might be complicated anyway with the GHA rebase approach, so I think we'd just have to rebase the PR on the branch it depends on?

Reply:
@stevenhorsman changes applied